media/js/src/filters/AssetFilter.jsx

```diff
@@ -252,9 +252,10 @@ export default class AssetFilter extends React.Component {
                             </div>
                         </div>
                         <div className="form-group col-md-2">
-                            <label htmlFor="react-select-2-input">Owner</label>
+                            <label htmlFor="owner-filter">Owner</label>
```

Comment:
Also - keep in mind that messing around with the id's and `for` attributes may not even be necessary. The way I prefer to associate labels with inputs is just wrap the label around the input. This is a completely acceptable, accessible way of using the html: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Reply:
@nikolas wrapping the label around the Select component warps the styling a bit :/

Reply:
Oh okay - yeah just doing it this way is probably easier then.